tikzplotlib/_text.py

```diff
@@ -9,18 +9,32 @@ def draw_text(data, obj):
     content = []
     properties = []
     style = []
+    ff = data["float format"]
     if isinstance(obj, mpl.text.Annotation):
-        _annotation(obj, data, content)
+        pos = _annotation(obj, data, content)
+    else:
+        pos = obj.get_position()
 
-    # 1: coordinates
-    # 2: properties (shapes, rotation, etc)
-    # 3: text style
-    # 4: the text
-    #                   -------1--------2---3--4--
```

Comment:
No idea what was intended by this comment, but it seems to have lost its usefulness